app/components/header_component.html.slim

```diff
@@ -7,13 +7,19 @@
     #service-name
       = link_to service_name, service_url, class: 'dfe-header__link dfe-header__link--service'
     #content-header.dfe-header__content.dfe-header-f-dtonly.dfe-header-f-toplinks
-      .dfe-header-f-top-links
-        / ul.dfe-header__action-links
-        /   - action_links.each do |item|
-        /     = item
       .dfe-header__menu
         button#toggle-menu.dfe-header__menu-toggle aria-controls='header-navigation' aria-expanded='false' data-action='click->reveal#toggle'
           | Menu
+    / This logic has been commented out as it is not required currently
```

Comment:
Could you check that this way of commenting doesn't result in visible HTML comments and use an alternative approach if it does please. Would a format like the following separated by blank lines read any better.

```

-# original header template
<their code>

-# custom service template
<our code>

```

Reply:
This comment isn't shown when inspecting where it is used. I will update this so it is more readable though 👍